lib/web_ui/lib/src/engine/compositor/image_filter.dart

```diff
@@ -10,7 +10,7 @@ part of engine;
 class SkImageFilter implements ui.ImageFilter {
   js.JsObject skImageFilter;
 
-  SkImageFilter.blur({double sigmaX = 0.0, double sigmaY = 0.0}) {
+  SkImageFilter.blur({this.sigmaX = 0.0, this.sigmaY = 0.0}) {
```

Comment:
Can you please leave a TODO with a link to an issue for us to implement `.matrix` in `SkImageFilter`? Should be straightforward to implement in the CanvasKit backend.

Reply:
There are SkImageFilter and EngineImageFilter both and they are constructed from ui.ImageFilter. Shouldn't there be a TODO in the public-facing class instead? Is this something that will really fall through the cracks? None of the classes implement the .matrix() constructor and the public facing class throws a hard (unconditional) exception if you call it...

Reply:
Are we fine without a TODO since ImageFilter.matrix() isn't implemented anywhere in web_ui and already has a "guard exception throw" in the main class?

Reply:
Between a TODO and a GitHub issue, the GitHub issue would be most useful.

Reply:
https://github.com/flutter/flutter/issues/45213